SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/DeploymentLauncher/DeploymentLauncher.cs

```diff
@@ -356,8 +365,8 @@ private static int CreateDeployment(string[] args)
         private static int StopDeployments(string[] args)
         {
             var projectName = args[1];
-
-            var deploymentServiceClient = DeploymentServiceClient.Create();
+            var regionCode = args[6];
```

Comment:
Why aren't we using arg[2] ?

Reply:
Also need to update usage (in `ShowUsage`) and argument number checking in `Main`